aten/src/ATen/CPUGeneratorImpl.cpp

```diff
@@ -94,6 +94,14 @@ void CPUGeneratorImpl::set_current_seed(uint64_t seed) {
   engine_ = mt19937(seed);
 }
 
+/**
+ * Manually seeds the engine with the seed input
+ * See Note [Acquire lock when using random generators]
+ */
+void CPUGeneratorImpl::set_offset(uint64_t offset) {
+  AT_ERROR("CPU Generator does not use offset");
```

Comment:
TORCH_CHECK

Reply:
What do you mean TORCH_CHECK? Here, my intent is - "you should not be here".
Earlier I was thinking - `throw std::runtime_error("not implemented");`

Reply:
`TORCH_CHECK` is something we use when user provides invalid input, we should deprecate AT_ERROR and use TORCH_CHECK in its place

Reply:
Ah, I see. Something like `TORCH_CHECK(false, ...)`. Sounds good.

Reply:
Yes AT_ERROR is a deprecated macro. TORCH_CHECK and TORCH_INTERNAL_ASSERT should be used.